maps/event/burzsia_station/burzsia_station.dm

```diff
@@ -0,0 +1,30 @@
+/datum/map/event/burzsia_station
+	name = "HICS Etna"
+	full_name = "HICS Etna"
+	path = "event/burzsia_station"
+	lobby_icons = list('icons/misc/titlescreens/sccv_horizon/sccv_horizon.dmi', 'icons/misc/titlescreens/aurora/synthetics.dmi', 'icons/misc/titlescreens/aurora/tajara.dmi', 'icons/misc/titlescreens/aurora/vaurca.dmi')
+	lobby_transitions = FALSE
+
+	allowed_jobs = list(/datum/job/visitor, /datum/job/xo, /datum/job/captain, /datum/job/chief_engineer, /datum/job/consular, /datum/job/cmo, /datum/job/hos, /datum/job/representative, /datum/job/operations_manager, /datum/job/rd)
```

Comment:
are the head staff jobs being here intentional? like are these for horizon crew?

Reply:
dm'd